packages/scripts/src/application.ts

```diff
@@ -233,6 +235,28 @@ export class Application {
         };
     }
 
+    public async remote({ port: preferedPort }: IRunOptions = {}) {
+        if (!process.send) {
+            throw new Error('Cant launch remote from main process');
+        }
+
+        await this.loadEngineConfig();
+        const { socketServer, close, port } = await this.launchHttpServer(preferedPort);
+        const forkedProcess = new ForkedProcess(process);
+        createCommunication(forkedProcess, socketServer, { port, onClose: () => close() });
+        forkedProcess.postMessage({ id: 'init' });
+    }
+
+    private async loadEngineConfig() {
+        const engineConfig = await fs.promises.findClosestFile(this.basePath, 'engine.config.js');
+        if (engineConfig) {
+            const { require: requiredModules } = await import(engineConfig);
```

Comment:
put in try/catch, with catch saying "failed evaluating config file: ${configFilePath}", e

add interface for the config file itself... (handling of optional values)